llm/server.go

```diff
@@ -54,8 +54,10 @@ type llmServer struct {
 	options     api.Options
 	numParallel int
 	modelPath   string
-	modelLock   sync.Mutex   // Temporary until we switch fully to Go server
-	model       *llama.Model // If non-nil, the runner is a new Go server
+
+	// Only one flavor will be loaded depending on the runner type
+	oldModel      *llama.Model
```

Comment:
```suggestion
	llamaModel      *llama.Model
```

Referring to any llama.cpp code as `llama` is a pattern that might be helpful here vs `old`, as there could be new models supported via llama.cpp